api-client/src/camera/types.ts

```diff
@@ -4,8 +4,20 @@ export interface CameraData {
   errorRecoveryCameraEnabled: boolean
 }
 
+export interface CameraImageSettings {
+  cameraId: CameraId
+  resolution?: [number, number]
+  zoom?: number
+  contrast?: number
+  brightness?: number
+  saturation?: number
+  pan?: [number, number]
+}
+
 export interface CreateCameraData {
   data: CameraData
 }
 
+export type CameraId = string
```

Comment:
```suggestion
export type CameraId = 'ot_system_camera'
```
I think this type should live in shared data, not api-client. `shared-data/js/types.ts` specfically.